cmd/root.go

```diff
@@ -93,15 +93,15 @@ func createServerCommand(system *core.System) *cobra.Command {
 	return &cobra.Command{
 		Use:   "server",
 		Short: "Starts the Nuts server",
-		RunE: func(cmd *cobra.Command, args []string) error {
+		Run: func(cmd *cobra.Command, args []string) {
 			// Load all config and add generic options
 			if err := system.Load(cmd.Flags()); err != nil {
-				return err
+				logrus.WithError(err).Error("could not start the server")
+				return
 			}
 			if err := startServer(cmd.Context(), system); err != nil {
-				return err
+				logrus.WithError(err).Error("could not start the server")
```

Comment:
Make it fatal, otherwise the exit code will just be 0 (indicating all is fine)
```suggestion
				logrus.WithError(err).Fatal("Could not start the server")
```